ocaml/doc/README.md

```diff
@@ -0,0 +1,11 @@
+# A note on generating locally the API reference
+
+Run `make doc`. The output is in `_build/ocaml/doc/markdown`.
+
+To view the markdown properly you will need to install `graphviz` so as to
+convert the dot files to images. To generate a pdf in addition, you will need to
+install `pandoc`.
```

Comment:
Does pdflatex requre explicit mention? I thought it was installed automatically as a pandoc dependency?